src/twisted/python/compat.py

```diff
@@ -429,7 +435,9 @@ def intToBytes(i: int) -> bytes:
     return b"%d" % (i,)
 
 
-def lazyByteSlice(object, offset=0, size=None):
+def lazyByteSlice(
+    object: Any, offset: int = 0, size: Optional[int] = None
```

Comment:
```python
    object: ReadableBuffer, offset: int = 0, size: Optional[int] = None
```

memoryview takes a ReadableBuffer https://github.com/python/typeshed/blob/16ae4c61201cd8b96b8b22cdfb2ab9e89ba5bcf2/stdlib/builtins.pyi#L608

defined, https://github.com/python/typeshed/blob/16ae4c61201cd8b96b8b22cdfb2ab9e89ba5bcf2/stdlib/_typeshed/__init__.pyi#L171

Reply:
Ah, nice find. The typeshed file says it's ok to use its definition from outside of typeshed but it doesn't guarantee stability — do you think it's better to copy the definition out into Twisted (meaning, Twisted's definition will become silently out of date if Python changes), or reference typeshed (meaning, Twisted may fail to typecheck if `_typeshed.py` changes)? I'm inclined to the latter, because it means that breakage would be immediately visible instead of hidden. (And typeshed doesn't have so much code churn that I would expect the definition to change soon.)

Reply:
Addressed in 8f483dc I think.